src/serialize.js

```diff
@@ -91,17 +96,19 @@ const build = ({
     }
 
     // Trouble primitives
-    if (_.isNaN(val)) return 'NaN';
-    if (val === Infinity) return 'Infinity';
-    if (val === -Infinity) return '-Infinity';
-    if (1 / val === -Infinity) return '-0';
-    if (typeof val === 'symbol') return val.toString();
+    if (_.isNaN(val)) {return 'NaN';}
```

Comment:
```suggestion
    if (_.isNaN(val)) {
      return 'NaN';
    } else if (...) { 
      ...
    } ...
```
Since each of these return these should probably be `else if` statements.